SRC/css-grids/css-grids.css

```diff
@@ -0,0 +1,59 @@
+.mainContainer{
```

Comment:
before `.mainContainer` add some style to your body so mainContainer takes up the whole screen:
`body {`
    `height: 100vh;`
   `margin: 0;`
`}`